src/plugins/ncr/components/with-node-screen/index.js

```diff
@@ -26,13 +27,21 @@ export default function withNodeScreen(Screen, config) {
     const { editMode, nodeRef, qname, label, tab, urls } = useParams(props, config);
     const { node, refreshNode, isRefreshing, setNode, pbjxError } = useNode(nodeRef, true);
     const canUpdate = policy.isGranted(`${qname}:update`);
+    const isLocked = node?.get('is_locked');
 
     useEffect(() => {
       if (editMode && !canUpdate) {
         navigate(urls.viewMode);
       }
     }, [editMode, canUpdate]);
 
+    useEffect(() => {
```

Comment:
@mtakla16 toast isn't needed here.  i'd make the useeffect dependencies just the node and policy